tests/examples/HHClientLinter/invalid_null_check.hack.expect

```diff
@@ -1,7 +1,7 @@
 [
     {
-        "blame": null,
-        "blame_pretty": null,
+        "blame": "cannot_be_null is null",
+        "blame_pretty": "cannot_be_null is null",
         "description": "Invalid null check: This expression will always return `false`.\nA value of type `int` can never be null."
     }
 ]
```

Comment:
(separate PR) It's probably worth adding range information to LinterTestTrait::getErrorAsShape() so that these tests also make sure the the information provided to IDEs is accurate. This will require updating many of the existing tests.